components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -1472,10 +1472,10 @@ var roleMappinRowID = -1;
                     			 %>
                     			 <tr>
                     			 <td>
-                    			 	<input name="req_path_auth" id="req_path_auth" type="hidden" value="<%=reqAth.getName()%>" />
-                    			 	<input name="req_path_auth_<%=reqAth.getName()%>" id="req_path_auth_<%=reqAth.getName()%>" type="hidden" value="<%=reqAth.getName()%>" />
+                    			 	<input name="req_path_auth" id="req_path_auth" type="hidden" value="<%=Encode.forHtmlAttribute(reqAth.getName())%>" />
+                    			 	<input name="req_path_auth_<%=reqAth.getName()%>" id="req_path_auth_<%=Encode.forHtmlAttribute(reqAth.getName())%>" type="hidden" value="<%=Encode.forHtmlAttribute(reqAth.getName())%>" />
```

Comment:
Use Encode.forHtmlAttribute for req_path_auth_<%=reqAth.getName()%> as below
req_path_auth_<%=Encode.forHtmlAttribute(reqAth.getName())%>